about.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
What does that do? :-)

Reply:
since it's not linked anywhere Sphinx got mad and warned, this silences that warning. I want the page so previous links to it continue to work, since we can't add more redirects